services/lgtm/lgtm-base.js

```diff
@@ -25,8 +31,17 @@ module.exports = class LgtmBaseService extends BaseJsonService {
     return { label: 'lgtm' }
   }
 
-  async fetch({ user, repo }) {
-    const url = `https://lgtm.com/api/v0.1/project/g/${user}/${repo}/details`
+  static get pattern() {
+    return ':host/:user/:repo'
```

Comment:
Thanks for the updates! We should still make this route pattern more restrictive to only allow the valid hosts we will accept. The only option we'll receive from existing badges is `g` which we can handle via our `redirector` functionality (details [here](https://github.com/badges/shields/pull/3362#discussion_r279043952) for reference).